assets/dev/scss/frontend/_lightbox.scss

```diff
@@ -68,11 +68,9 @@
 		}
 	}
 
-	.swiper- {
-
-		&container {
-			height: 100%;
-		}
+	.swiper,
+	.swiper-container {
+		height: 100%;
```

Comment:
Does this not cause BC issues?

Reply:
@hein-obox What do you mean?

Reply:
In the old version the '.swiper' class has a height of 100%.

Reply:
Slack

Reply:
Discussion 'solved' on Slack.